src/theory/sep/theory_sep.cpp

```diff
@@ -412,7 +412,7 @@ void TheorySep::reduceFact(TNode atom, bool polarity, TNode fact)
     }
     else if (satom.getKind() == SEP_PTO)
     {
-      Node ss = nm->mkNode(SINGLETON, satom[0]);
+      Node ss = nm->mkSingleton(satom[0].getType(), satom[0]);
```

Comment:
Note these uses of type are not safe, e.g. if satom[0] is intended to be a Real but is 1.0.  I will address this after this PR, as this may require more refactoring to TheorySep to know which type to use.

Reply:
Added issue https://github.com/CVC4/cvc4-projects/issues/230